lib/widgets/loginform.dart

```diff
@@ -115,47 +117,13 @@ class _LoginFormState extends State<LoginForm> {
           mainAxisAlignment: MainAxisAlignment.center,
           crossAxisAlignment: CrossAxisAlignment.center,
           children: <Widget>[
-            Padding(
-              padding: const EdgeInsets.symmetric(vertical: 20.0),
-              child: Image.asset('assets/parkr_logo.png'),
+            const Logo(),
+            EmailField(
+              emailController: emailCtrl,
```

Comment:
I mean, this seems fine to me. I'm not sure. If it works, I would rock it. Could always put the control within the EmailField and have like a getText() method on it, too, if that feels better